### PR TITLE
New OPA website

### PR DESCRIPTION
**Pull request by lucperkins**
This PR creates a new OPA website that uses [Hugo](https://gohugo.io) as a site generator (rather than GitBook) and publishes the site on [Netlify](https://netlify.com). A preview of the new site can be viewed at https://openpolicyagent-dev.netlify.com. It also includes corresponding changes to the current build pipeline (removing bits of that pipeline that are no longer necessary with automated site deployment).

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
-### Prerequisites
+## Prerequisites
 
 If you want to build and serve the site locally, you need the following packages
 installed on your system:
 
-- npm
-- [gitbook](https://github.com/GitbookIO/gitbook)
+- [npm](https://npmjs.org)
+- The [Hugo](https://gohugo.io/getting-started/installing/) static site generator
```
Comment (tsandall) [inline docs/README.md]: I installed Hugo v0.54.0 but I'm getting this error:

```
$ make serve
hugo server \
	--buildDrafts \
	--buildFuture \
	--disableFastRender
Building sites … ERROR 2019/03/21 08:57:41 error: failed to transform resource: TOCSS: failed to transform "sass/style.sass" (text/x-sass): this feature is not available in your current Hugo version
Total in 330 ms
Error: Error building site: logged 1 error(s)
Makefile:5: recipe for target 'serve' failed
make: *** [serve] Error 255
```

File: docs/README.md [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
 
-### Build and preview the docs locally
+Once npm and Hugo are installed on your system, run `make setup` to install the
```
Comment (tsandall) [inline docs/README.md]: Nit: Would be good to explicitly state that people must install Hugo themselves (and any version requirements)

File: docs/README.md (hunk 1 above)
Comment (tsandall) [reply docs/README.md]: I was able to resolve this by cloning the Hugo repo and building from source with `--tags extended`:

```
go build --tags extended
```

Note, I had to upgrade the Go version on my development machine to 1.12.

@lucperkins could you figure out how to make this step a bit more reproducible?

Comment (lucperkins) [reply docs/README.md]: @tsandall Yeah, I can add OS-specific docs around that.

**Comment by lucperkins** [comment]
@tsandall Okay, I think that these initial concerns have been addressed if you'd like to take a look. I'm still hacking away at a few things, so hold off on merging for the moment.

**Comment by tsandall** [comment]
@lucperkins I'm running into the following error when I try to `make serve` or run the docker image described in the docs README:

```
$ make serve
hugo server \
	--buildDrafts \
	--buildFuture
Building sites … ERROR 2019/03/22 16:19:17 error: failed to transform resource: SCSS processing failed: file "stdin", line 8, col 1: File to import not found or unreadable: bulma/sass/utilities/initial-variables. 
Total in 261 ms
Error: Error building site: logged 1 error(s)
Makefile:5: recipe for target 'serve' failed
make: *** [serve] Error 255
```

Any ideas?

**Comment by lucperkins** [comment]
@tsandall Ah, my bad. You’ll need to npm install first. I’ll update the README to reflect that later today.

**Comment by lucperkins** [comment]
@tsandall Okay, those instructions have been updated

**Comment by tsandall** [comment]
@lucperkins I was able to successfully run through the docs/README instructions after your change. The only thing I noticed was that @timothyhinrichs's cheatsheet didn't make it into the TOC. I can add that in after we merge though. Let me know once the Netlify configuration is sorted out.

**Comment by lucperkins** [comment]
@tsandall I added the cheatseet to the new system. Just needed to attach a few metadata items. So that's all set.

**Comment by tsandall** [comment]
Boom. :100:

**Pull request merged**